src/pokete/classes/fight/attack_process.py

```
@@ -67,14 +68,32 @@ def __call__(self, attacker: Poke, defender: Poke, attack: Attack,
 
             attacker.backup_hp()
             defender.backup_hp()
+
+            # Resilience Blessing: 20% chance for Honored Specialists in RELIEF mode to survive lethal hits at 1HP
+            if (defender.player and dm.get_pacing_mode() == "RELIEF" and
+                dm.get_favor() == "Honored Specialist" and n_hp >= defender.hp > 0):
+                if random.random() < 0.2:
```

Ruff (0.14.14)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)